src/catkin_pkg/package_templates.py

```diff
@@ -41,6 +41,8 @@
 from catkin_pkg.package import Package
 from catkin_pkg.package import PACKAGE_MANIFEST_FILENAME
 from catkin_pkg.package import Person
+from catkin_pkg.cmake import get_metapackage_cmake_template_path
+from catkin_pkg.cmake import configure_file
```

Comment:
The order in the cmake.py file looks fine to me: https://github.com/ros-infrastructure/catkin_pkg/blob/fc10bd1597697e7138477823907de65e74201ec5/src/catkin_pkg/cmake.py#L34-L37: `_` before `o` before `r`.